easybuild/easyblocks/h/hadoop.py

```diff
@@ -43,6 +43,7 @@ def extra_options():
         """Custom easyconfig parameters for Hadoop."""
         extra_vars = {
             'build_native_libs': [False, "Build native libraries", CUSTOM],
+            'parallel': [1, "Build in parallel with more threads", CUSTOM],
```

Comment:
Why fix this to `1` here? `parallel` can be set in all easyconfigs, just picking it up in the easyblock is sufficient